src/services/course-service.ts

```diff
@@ -65,7 +50,7 @@ export const CourseService = {
       })
     })
   },
-  editCourseQuery: async (id: string, data: CourseForm) => {
+  editCourse: async (id: string, data: CourseForm) => {
```

Comment:
```suggestion
  editCourse: (id: string, data: CourseForm) => {
```

Why do you need ```async``` here?